src/components/live/faq/FAQ.jsx

```diff
@@ -0,0 +1,24 @@
+import { ITEMS } from "@/data/FAQ";
+import { Accordion } from "react-bootstrap";
```

Comment:
this can be done without an accordion, check prior hackathon code, there have been implementations without accordion, Rosehack did it without I believe